about.html

```diff
@@ -22,24 +21,19 @@
 <h3>Applications</h3>
 
 <ul>
-<li>PcManFm-qt - File Manager</li>
-<li>Lximage-qt - Image Viewer</li>
-<li>QTerminal - Terminal Emulator</li>
-<li>Qps - Process Viewer</li>
-<li>Screengrab - Screen Capturer</li>
-<li>LXQt-archiver - Archive Manager</li>
-<li>LXQt-runner - Application launcher and calculator</li>
+<li>lxqt-runner — Launch applications and do calculations</li>
+<li>PCManFM-Qt — Browse files</li>
+<li>LXImage-Qt — View images</li>
+<li>QTerminal — Use the terminal</li>
+<li>ScreenGrab — Capture screenshots</li>
+<li>LXQt Archiver — Manage archives</li>
+<li>Qps — Manage processes</li>
 </ul>
 <br/>
 
-<p>LXQt is available for all major Linux and BSD distributions so you may just try it
-out on your system or in a VM.<br>
-More information about installing can be found in the
-<a href="https://github.com/lxqt/lxqt/wiki/">LXQt GitHub Wiki</a>.
+<p>Try LXQt in most Linux or BSD distributions.<br>
+Learn to install it in the <a href="https://github.com/lxqt/lxqt/wiki/">LXQt wiki</a>.
 </p>
 <p>
-Historically, LXQt was the product of the merge between LXDE-Qt, an initial Qt
-flavour of LXDE, and Razor-qt, a project aiming to develop a Qt based desktop environment.<br>
-More on this topic can be found in the
-<a href="https://github.com/lxqt/lxqt/wiki/History">Wiki</a>.
+Read more about its history in the <a href="https://github.com/lxqt/lxqt/wiki/History">LXQt wiki</a>.
```

Comment:
Imperative again.

Reply:
On an "About" page — in contrast to a short reply or a page about instructions — we usually don't tell people what to do but what's available for them.